docs/guides/misc/vlans.md

```diff
@@ -0,0 +1,71 @@
+# VLANs and virtual LAN interfaces
+
+This guide will help you configure Pi-hole to operate across defined virtual LAN interfaces on networks with VLANs configured.
+
+## Notes & Warnings
```

Comment:
Maybe you should add a note that this configuration only works with `dhcpcd`. Raspbian is goind (or already has) switched to `NetworkManager`